### ISSUE TITLE
Strange Characters Appear when using XTerm and Konsole

### ISSUE THREAD

**Issue by jtplaarj** [piccolomo/plotext#62]
Comparing the output of versions 3.1.3 and 4.1.0, the latter has some strange extra chars:

![Screenshot_20211210_113122](https://user-images.githubusercontent.com/22937107/145560135-eb560da5-ba29-434c-b4d5-88ccb6a2228a.png)

It looks like 3.1.3 produces the correct output.

**Comment by piccolomo** [comment]
Hi @jtplaarj ,

thanks a lot for the issue report. I have seen this problem only on Windows, and I assume that's your platform. I created the package under Ubuntu and I use Windows to test it, but less often. 

Any way I have tried to fix the problem with a new version (4.1.1). I have not upload it on PyPi, but only on GitHub. Do you have the possibility of kindly testing this new version and see if the problem is solved? Let me know if you can't or if you you need me to publish it on PyPi first.

Thanks a lot,
Savino

**Comment by jtplaarj** [comment]
Hi.

The screen shoot was on Debian. I do not have a Debian machine at hand, but I have tested it on Ubuntu 20.04.3 LTS and the results are similar. A screen shoot with 4.1.1 at the top, 4.1.0 bottom left and 3.1.3 bottom right:

![Screenshot_20211211_093222](https://user-images.githubusercontent.com/22937107/145670128-7a92da0b-3f31-4d4b-8fa7-6da1971449f9.png)

**Comment by piccolomo** [comment]
Hi @jtplaarj ,

I really don't understand what could be the issue. I only noticed that version 3.1.3 defined the escape codes in a different way. I have now used this old definition and it still works, but it worked also with the new escape codes, so the only way is for you to test this new version 4.1.2. 

Does it work?

Thanks

**Comment by jtplaarj** [comment]
It is still happening.... both my two tests have been using konsole. This may be one different point between your setup and mine.
I have tried with xterm and the result is similar.

**Comment by piccolomo** [comment]
Hi @jtplaarj ,

ok, i have tested xterm and you are right: it doesn't work.

I haven't investigated deeply the issue but from [here](https://unix.stackexchange.com/questions/280466/how-do-i-type-arbitrary-unicode-characters-in-xterm) it seems that xterm doesn't implement some unicode characters: in our case, the 3x2 mosaic markers like 🬤.

Anyway if you specify `marker = 'hd'` or `marker = 'sd' `inside the `plt.bar()` function, the plot works.

If in the meantime, you or someone else finds a workaround, it will be great.

Thanks

I could try to investigate this for next version.

**Comment by piccolomo** [comment]
Btw, the standard Ubuntu terminal seems the best choise.

**Comment by piccolomo** [comment]
> It is still happening.... 

any update?

**Comment by jtplaarj** [comment]
Hi,

The solutions you mention ```marker = 'hd'``` and ```marker = 'sd'```both work perfectly.

I have tested with [alacritty](https://github.com/alacritty/alacritty) and it works perfectly.

Maybe a section in the manual stating supported tty could be enough. Otherwise, detecting the console and using a set of different defaults could also be a solution.

Finding out what makes some Unicode char work and other no, may be quite complex.

**Comment by piccolomo** [comment]
Hi @jtplaarj,

thanks for reply. Here is the [new 4.1.3 version ](https://github.com/piccolomo/plotext) and this is what I have done: since it is almost impossible for me to discern which terminals renders well the 3x2 marker called `fhd`, I made the 2x2 ,marker called `hd` the default one for all terminal. 

The marker `fhd` could still be used but one needs to check if its terminal allows it and render thos characters well first.

I also changed the background color and added few other minor changes. If you are available in double testing it, it would be helpful and I should push the changes also on PyPi later. I shall close the issue at your reply.

Thanks
